ansible/logs.yml

```diff
@@ -27,11 +27,19 @@
 - hosts: all:!ansible
   serial: 1
   tasks:
+  - name: set host flag
+    set_fact:
+      docker_host_flag: "--host tcp://{{ ansible_host }}:{{ docker.port }}"
+    when: (whisk_version_name == "mac" or ansible_os_family == "Debian")
```

Comment:
will this work in our pgs?

Reply:
yes works on pgs

Reply:
confirmed that it did work on the distributed environment... but i'm a little puzzled why --- can you help me understand why this works on gnu/linux?

Reply:
because this`ansible_os_family == "Debian"` will be `true` in those environments

Reply:
Once we remove `docker-machine` this will look cleaner

Reply:
wondering if we can condition this on `!local`.

Reply:
ah missed some other places